airflow/www_rbac/api/experimental/endpoints.py

```diff
@@ -176,6 +177,42 @@ def task_instance_info(dag_id, execution_date, task_id):
     return jsonify(fields)
 
 
+@api_experimental.route(
+    '/dags/<string:dag_id>/dag_runs/<string:execution_date>/status',
```

Comment:
I think the `/status` is redundant and it should just point to /dag_runs/<exec_date>/